lms/envs/common.py

```diff
@@ -502,18 +502,26 @@
 
 ################################## DJANGO OAUTH TOOLKIT #######################################
 
+OAUTH2_DEFAULT_SCOPES = {
+    'read': _('Read access'),
+    'write': _('Write access'),
+    'email': _('Know your email address'),
+    # conform profile scope message that is presented to end-user
+    # to lms/templates/provider/authorize.html. This may be revised later.
```

Comment:
Nit: I'm not sure what this comment means and whether it's still valid.

Reply:
I will update the comment to something useful.